s5_subwords/runV3.sh

```diff
@@ -45,7 +45,8 @@ method='bpe'
 AUDIO=/data/asr/malromur/malromur2017/correct
 
 #samromur_root=/data/asr/samromur/samromur_ldc
-METADATA=/data/asr/malromur/malromur2017/malromur_metadata.tsv
+# METADATA=/data/asr/malromur/malromur2017/malromur_metadata.tsv
+METADATA=/home/derik/work/tools/normalize/malromur/normalized_files/malromur_metadata_subset.tsv
```

Comment:
If this is static, you should add it to the preprocessed directory like the samromur and samromur teen metadata files.

Reply:
I'm not sure I'm following. This file is only a temp file to speed up dev

Reply:
ok if it's only temp, no need to do anything